data/json/items/ammo/762x25.json

```diff
@@ -46,13 +46,25 @@
     "//": "Stopgap price of $1 per.  Anti-China prejudice in the pre-Cataclysm US may have made this ammo tougher to come by.",
     "description": "A subsonic cartridge derived from the 7.62x25mm, designed for silenced firearms.  It offers good armor penetration at the cost of slightly less damage.",
     "weight": 12,
-    "price": 5000,
     "count": 50,
     "relative": {
+      "price": 2000,
       "range": -1,
       "damage": -9,
       "pierce": 2,
-      "recoil": -45
+      "recoil": -270
+    }
+  },
+  {
+    "id": "reloaded_762_25hot",
+    "copy-from": "762_25",
+    "type": "AMMO",
+    "name": "reloaded 7.62x25 hot load",
```

Comment:
I guess you need to define plural name for this item, since now its plural name will be automatically defined as "reloaded 7.62x25 hot loads" (I didn't check it in the game but I expect to see this).